specification/synapse/resource-manager/readme.md

```diff
@@ -28,10 +28,20 @@ These are the global settings for the Azure Synapse Analytics API.
 description: Azure Synapse Analytics Management Client
 openapi-type: arm
 azure-arm: true
-tag: package-2021-03
+tag: package-2021-04
 generate-empty-classes: true
+modelerfour:
+  lenient-model-deduplication: true
 ```
 
+### Tag: package-2021-04
+
+These settings apply only when `--tag=package-2021-04` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-04'
+input-file:
+  - Microsoft.Synapse/stable/2021-04-04/kustoPool.json
```

Comment:
The input files should also contains those of 2021-03-01. It should look like below:

```
  - Microsoft.Synapse/stable/2021-04-04/kustoPool.json
  - Microsoft.Synapse/stable/2021-03-01/bigDataPool.json
  - Microsoft.Synapse/stable/2021-03-01/checkNameAvailability.json
  - Microsoft.Synapse/stable/2021-03-01/firewallRule.json
  - Microsoft.Synapse/stable/2021-03-01/integrationRuntime.json
  - Microsoft.Synapse/stable/2021-03-01/keys.json
  - Microsoft.Synapse/stable/2021-03-01/library.json
  - Microsoft.Synapse/stable/2021-03-01/operations.json
  - Microsoft.Synapse/stable/2021-03-01/privateEndpointConnections.json
  - Microsoft.Synapse/stable/2021-03-01/privateLinkResources.json
  - Microsoft.Synapse/stable/2021-03-01/privatelinkhub.json
  - Microsoft.Synapse/stable/2021-03-01/sqlPool.json
  - Microsoft.Synapse/stable/2021-03-01/sqlServer.json
  - Microsoft.Synapse/stable/2021-03-01/workspace.json
```